axon-framework/events/event-bus-and-event-store.md

```diff
@@ -393,14 +393,30 @@ If no explicit `eventSerializer` is configured, events are serialized using the
 
 ## Distributing Events
 
-The use of an `EventStore` provides a simple way of distributing events: by sharing the data source.
-This is the approach taken by [Axon Server](../../axon-server-introduction.md), for example.
+To distribute events between applications, it is paramount to know whether the applications belong to the same [bounded context](../../architecture-overview/ddd-cqrs-concepts.md#bounded-context).
```

Comment:
"paramount" might be over the top (pun intended :) ) here. Why not just say "important".

Reply:
Sounds good to if you ask me.